dali/test/python/test_pipeline.py

```diff
@@ -1718,3 +1718,11 @@ def test_return_constants():
     for i, o in enumerate(other):
         assert o.at(0) == types[i](42)
         assert o.at(0).dtype == types[i]
+
+def test_preserver():
+    pipe = dali.pipeline.Pipeline(1, 1, 0)
+    with pipe:
+        out = dali.fn.external_source(source=[[np.array([-0.5, 1.25])]], preserve = True)
+        res = dali.fn.resize(out, preserve = True)
+        pipe.set_outputs(out)
+    pipe.build()
```

Comment:
you don't run any iteration?

Reply:
It checks if build succeeds and args are properly consumed.